src/sentry/ingest/transaction_clusterer/tasks.py

```diff
@@ -91,3 +93,75 @@ def cluster_projects(projects: Sequence[Project]) -> None:
             metrics.incr(
                 "txcluster.cluster_projects.unclustered", amount=unclustered, sample_rate=1.0
             )
+
+
+@instrumented_task(
+    name="sentry.ingest.span_clusterer.tasks.spawn_span_cluster_projects",
+    queue="span.descs.clusterer",  # XXX(iker): we should use a different queue
+    default_retry_delay=5,
+    max_retries=5,
+    soft_time_limit=PROJECTS_PER_TASK * CLUSTERING_TIMEOUT_PER_PROJECT,
+    time_limit=PROJECTS_PER_TASK * CLUSTERING_TIMEOUT_PER_PROJECT + 2,
```

Comment:
I'm not sure what the code suggestion is.

Regarding the number of projects -- it doesn't depend for now, but it will in the future. I don't know when that future is, and the implementation cost is pretty much the same as I was copy-pasting the transaction clusterer implementation. Is this reasonable?